timeseries/setup.py

```diff
@@ -30,7 +30,7 @@
     "pytorch_lightning",  # version range defined in `core/_setup_utils.py`
     "transformers[sentencepiece]",  # version range defined in `core/_setup_utils.py`
     "accelerate",  # version range defined in `core/_setup_utils.py`
-    "gluonts==0.15.1",
```

Comment:
Shall we pin the version or introduce it as a lower, upper bound range?

Reply:
I'm planning to update to lower bound once `gluonts` is release (hopefully before AG 1.2.

Reply:
Updated as `gluonts` just released 0.16.0